lib/sparql.jison

```diff
@@ -89,8 +87,7 @@
   function toVar(variable) {
     if (variable) {
       var first = variable[0];
-      if (first === '?') return variable;
-      if (first === '$') return '?' + variable.substr(1);
+      if (first === '?' || first === '$') return Parser.factory.variable(variable.substr(1));
```

Comment:
condition will always be true

Reply:
I thought so too, but the code disagreed :) I can come back to check this later